ckanext/xloader/action.py

```diff
@@ -334,14 +335,13 @@ def xloader_status(context, data_dict):
         db.init(config)
         job_detail = db.get_job(job_id)
 
-        # timestamp is a date, so not sure why this code was there
+        # Attach time zone data to logs if needed
```

Comment:
This did not work and was re-commented out. 
Also time range is not working correctly on presentation but when looking at the metadata it is correct.